extras/common.sh

```diff
@@ -0,0 +1 @@
+../common.sh
```

Comment:
This won't work if you run from some PWD that's not extras, instead you could use `$SCRIPTDIR` like:

https://github.com/openshift-metalkube/dev-scripts/blob/master/demo/launch_demo_bmo.sh#L11

Reply:
SCRIPTDIR will not xists on kafka.sh context I've just create a BASEDIR pointing to base repo dir that should do the same

Reply:
Yeah I was suggesting you copy the same approach